ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -2212,6 +2247,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-ipi-virtual-media
+- as: vsphere-ipi-amd-f7-netobserv
```

Comment:
vsphere does not support multi payload yet, so we should keep it in amd ci yaml file.